driver-core/src/test/unit/com/mongodb/ConnectionStringSpecification.groovy

```diff
@@ -279,7 +279,7 @@ class ConnectionStringSpecification extends Specification {
         new ConnectionString('mongodb://jeff@localhost/?' +
                            'authMechanism=GSSAPI')            | createGSSAPICredential('jeff')
         new ConnectionString('mongodb://jeff:123@localhost/?' +
-                           'authMechanism=PLAIN')             | createPlainCredential('jeff', 'admin', '123'.toCharArray())
+                           'authMechanism=PLAIN')             | createPlainCredential('jeff', '$external', '123'.toCharArray())
```

Comment:
This is possibly a breaking change for users that used PLAIN without declaring a database or an authSource.

Reply:
Worth adding to the release notes to the Upgrading section.

Reply:
Done!